kakao/src/main/kotlin/com/agoda/kakao/Builders.kt

```diff
@@ -173,6 +173,15 @@ class ViewBuilder {
         viewMatchers.add(ViewMatchers.withContentDescription(description))
     }
 
+    /**
+     * Matches the view with given content description
+     *
+     * @param resourceId Resource id of content description to match
+     */
+    fun withContentDescription(resourceId: Int) {
```

Comment:
Could you please annotate with `@StringRes` as well?

Reply:
Done!